pkg/spanconfig/target.go

```diff
@@ -248,3 +266,15 @@ func TargetsFromProtos(protoTargets []roachpb.SpanConfigTarget) ([]Target, error
 	}
 	return targets, nil
 }
+
+// TestingEntireSpanConfigurationStateTargets returns a list of targets which
+// can be used to read the entire span configuration state. This includes all
+// span configurations installed by all tenants and all system span
+// configurations, including those installed by secondary tenants.
+func TestingEntireSpanConfigurationStateTargets() []Target {
+	return Targets{
+		Target{
+			span: keys.EverythingSpan,
```

Comment:
Should we instead break this down to system targets and span targets instead of implicitly relying on the system target encodings? If we care about being explicit, we should add assertions in KV accessor to not allow spans to straddle the "special" keyspace, or explicitly drop those records from the results.